ruby/files_and_serialization/project_event_manager.md

```diff
@@ -1,6 +1,6 @@
 ### Introduction
 
-Please note this tutorial has been adapted from The Turing School's and Jump Start Lab's [Event Manager](http://tutorials.jumpstartlab.com/projects/eventmanager.html) and updated to use GoogleCivic API
+Please note this tutorial has been adapted from The Turing School's and Jump Start Lab's [Event Manager]( https://web.archive.org/web/20240908085655/https://tutorials.jumpstartlab.com/projects/eventmanager.html) and updated to use GoogleCivic API
```

Comment:
```suggestion
Please note this tutorial has been adapted from The Turing School's and Jump Start Lab's [Event Manager](https://web.archive.org/web/20240908085655/https://tutorials.jumpstartlab.com/projects/eventmanager.html) and updated to use GoogleCivic API
```
Just one small suggestion to get rid of the leading whitespace in the link's href.